accord-core/src/main/java/accord/utils/SortedArrays.java

```diff
@@ -1793,4 +1794,29 @@ private static void swap(int[] values, int i, int j)
         values[i] = values[j];
         values[j] = t;
     }
+
+    public static <T extends Comparable<? super T>> BitSet toBitSet(SortedArrays.SortedArrayList<T> src,
+                                                                    SortedArrays.SortedArrayList<T> subset)
+    {
+        BitSet bitSet = new BitSet(src.size());
+        for (int i = 0; i < src.size(); i++)
+        {
+            if (subset.contains(src.get(i)))
+                bitSet.set(i);
+        }
+        return bitSet;
+    }
+
+    public static <T extends Comparable<? super T>> SortedArrays.SortedArrayList<T> fromBitSet(SortedArrays.SortedArrayList<T> src,
+                                                                                               BitSet bitSet,
```

Comment:
More consistent to use `SimpleBitSet` here as well, since this is what this codebase uses primarily